src/main/main.cpp

```diff
@@ -302,7 +313,7 @@ main(int argc, char* const* argv)
         Logging::setLogLevel(logLevel, nullptr);
 
         cfg.REBUILD_DB = newDB;
-        cfg.FORCE_SCP = forceSCP;
+        cfg.FORCE_SCP = (forceSCP != nullptr);
```

Comment:
this is a break of contract: code depending on config.forceSCP expects it to be true iif forceSCP is set globally.